src/Chainweb/Version.hs

```diff
@@ -777,8 +777,8 @@ twentyChainUpgrade
     -> ChainId
     -> BlockHeight
     -> Bool
-twentyChainUpgrade Mainnet01 _ 745000 = True
-twentyChainUpgrade Development _ 150 = True
+twentyChainUpgrade Mainnet01 _ h = h == 745000
+twentyChainUpgrade Development _ h = h == 150
```

Comment:
Could this be derived from `chainGraphs`? If not, does `chainGraphs` put some constraints on this function? If so, could you please add a comment or even a test case?

Reply:
we could probably re-instrument the way we determine block heights with `chainGraphs`, but that's a separate PR addressing a separate issue.